moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -1737,49 +1734,59 @@ bool PlanningScene::processCollisionObjectAdd(const moveit_msgs::CollisionObject
   if (object.operation == moveit_msgs::CollisionObject::ADD && world_->hasObject(object.id))
     world_->removeObject(object.id);
 
-  const Eigen::Isometry3d& object_frame_transform = getFrameTransform(object.header.frame_id);
+  const Eigen::Isometry3d& world_to_object_header_transform = getFrameTransform(object.header.frame_id);
+  Eigen::Isometry3d header_to_pose_transform;
+  PlanningScene::poseMsgToEigen(object.pose, header_to_pose_transform);
+  const Eigen::Isometry3d object_frame_transform = world_to_object_header_transform * header_to_pose_transform;
+
+  ROS_WARN_STREAM("object.header.frame_id = " << object.header.frame_id);
+  ROS_WARN_STREAM("world_to_object_header_transform = " << world_to_object_header_transform.matrix());
+  ROS_WARN_STREAM("header_to_pose_transform = " << header_to_pose_transform.matrix());
+  ROS_WARN_STREAM("object_frame_transform = " << object_frame_transform.matrix());
```

Comment:
These should be ROS_WARN_STREAM_NAMED(LOGNAME, ...)

Reply:
Those shouldn't be in there at all, they're just my debugging prints 😱